src/Hydration.js

```diff
@@ -1,36 +1,27 @@
-class Hydration {
+import StatsParent from "./StatsParent";
+
+class Hydration extends StatsParent {
   constructor(hydrationData, userID) {
+    super()
     this.hydrationData = hydrationData;
     this.userID = userID;
   }
 
-  findUser() {
-    return this.hydrationData.filter(user => user.userID === this.userID);
-  }
-
-  returnWeek() {
-    var specificUser = this.findUser()
-    return [...specificUser].splice(-7).map(day => day.date);
+  returnHydrationWeek(weekNum, data, id) {
+    return this.returnWeek(weekNum, data, id)
```

Comment:
Interesting way of doing this!  Totally keeps you from having to refactor the DOM.  One question I have for @brittanystoroz and @khalidwilliams  - for the class I refactored, I took the functions that inherited from the parent class out completely.  Is it ok that Colin and I are doing it different ways or do you all want consistency across the project?